extension/pom.xml

```diff
@@ -0,0 +1,46 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.bot</groupId>
+        <artifactId>build-reporting-maven-extension-parent</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>build-reporting-maven-extension</artifactId>
+    <name>Build Reporting Maven Extension</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus.bot</groupId>
+            <artifactId>build-reporting-maven-extension-model</artifactId>
```

Comment:
I split the model into a separate artifact so that I can easily reuse it in the bot.